tests/utils/deployment/deployment.go

```diff
@@ -191,3 +192,22 @@ func RedefineWithNodeAffinity(deployment *v1.Deployment, key string) *v1.Deploym
 
 	return deployment
 }
+
+func RedefineWithLivenessProbe(deployment *v1.Deployment) *v1.Deployment {
+	if len(deployment.Spec.Template.Spec.Containers) > 0 {
+		for index := range deployment.Spec.Template.Spec.Containers {
```

Comment:
You don't need the `if len(deployment.Spec.Template.Spec.Containers) > 0`, as the `range` function won't iterate when Containers is nil or empty.